src/scss/style.scss

```diff
@@ -1,14 +1,123 @@
+// Variables
+$orange: #F08080;
+// test
+$gray: #ECEDE9;
+$black: #000000;
+$white: #ffffff;
+
+
+
+// General styling
 body {
-  background-color: pink;
-  font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
-  height: 2000px;
+  font-family: 'Noto Serif JP', sans-serif;
+  color: $black;
+  margin: 0;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h5,
+h6,
+p {
+  margin: 0;
+}
+
+img {
+  height: auto;
+  display: block;
+  align-self: center;
+}
+
+h1 {
+  text-align: center;
+  font-size: 3em;
+  padding: 70px 40px;
+}
+
+
+
+
+// Section styling
+h2 {
+  font-weight: 900;
+  font-size: 2em;
```

Comment:
`font-size`s and `font-weight`s also like to be put into variables for consistency. :)